### PR TITLE
N.O. pregnancy model 5-8 V&V summary

### PR DESCRIPTION
**Pull request by alibow**
Sorry for the delay! Let me know if there are any questions about next steps for the two identified issues. Thanks!

### CODE REVIEW

File: docs/source/models/concept_models/vivarium_nutrition_optimization/pregnancies/concept_model.rst [hunk 1]
```diff
@@ -588,6 +588,22 @@ Specific outputs for specific models are specified in the following section.
...
+        * `Interactive sim to check that joint anemia/BMI exposure does not change over time <https://github.com/ihmeuw/vivarium_research_nutrition_optimization/blob/data_prep/verification_and_validation/pregnancy_model/model_7_interactive_sim.ipynb>`_
+  * - 6.0
+    - Verify:
+        * Scenario-specific intervention coverage
+        * Intervention impacts on hemoglobin exposure
+    - * `Intervention coverage looks good by scenario <https://github.com/ihmeuw/vivarium_research_nutrition_optimization/blob/data_prep/verification_and_validation/pregnancy_model/model_6_intervention_coverage.ipynb>`_, and `confirmed to be appropriately targeted to BMI exposure in the interactive sim <https://github.com/ihmeuw/vivarium_research_nutrition_optimization/blob/data_prep/verification_and_validation/pregnancy_model/model_7_interactive_sim.ipynb>`_
+      * Draw-level uncertainty in intervention effect on hemoglobin erroneously applied as individual-level stochastic uncertainty, as shown in the interactive sim linked above
```
Comment (patricktnast) [inline docs/source/models/concept_models/vivarium_nutrition_optimization/pregnancies/concept_model.rst:602]: The logic makes sense here--the reason I sampled this at the individual level is that is what IV Iron was doing, unless I'm understanding the code incorrectly. Just added this in the (forthcoming) bugfix.

File: docs/source/models/concept_models/vivarium_nutrition_optimization/pregnancies/concept_model.rst [hunk 2]
```diff
@@ -605,6 +621,14 @@ Specific outputs for specific models are specified in the following section.
...
     - TBD
+  * - Uncertainty in the intervention impact on hemoglobin applied as individual heterogeneity rather than draw-level uncertainty
+    - Effect size uncertainty represents statistical uncertainty about the estimation of the population mean difference rather than a measure of standard deviation in measured individual responses to the intervention
+    - Update how we handle uncertainty for this parameter
+    - Bugfix run
+  * - No baseline calibration of IFA effect on hemoglobin
+    - IFA effect on hemoglobin should be adjusted for baseline coverage as described in the intervention model document so that those with no baseline coverage experience a hgb change equal to -hgb_shift * baseline_coverage and those with baseline coverage experience a hgb change equal to hgb_shift * (1 - baseline_coverage)
+    - Update how we handle baseline coverage
```
Comment (patricktnast) [inline docs/source/models/concept_models/vivarium_nutrition_optimization/pregnancies/concept_model.rst:630]: Whoops, I basically shot right past that code snippet in the docs. This baseline adjustment makes sense, but now I'm a bit confused about how the interventions "stack" with respect to the baseline scenario.

File: docs/source/models/concept_models/vivarium_nutrition_optimization/pregnancies/concept_model.rst (hunk 1 above)
Comment (alibow) [reply docs/source/models/concept_models/vivarium_nutrition_optimization/pregnancies/concept_model.rst:602]: This is looking good in the latest run, removed the V&V issue in the latest commit

File: docs/source/models/concept_models/vivarium_nutrition_optimization/pregnancies/concept_model.rst (hunk 2 above)
Comment (alibow) [reply docs/source/models/concept_models/vivarium_nutrition_optimization/pregnancies/concept_model.rst]: This is looking good in the latest run, removed the V&V issue in the latest commit

File: docs/source/models/concept_models/vivarium_nutrition_optimization/pregnancies/concept_model.rst [hunk 3]
```diff
@@ -605,13 +664,17 @@ Specific outputs for specific models are specified in the following section.
...
+    - Update ordering of random.choice call so that "other" outcomes are not in the middle
+    - For next model run
 
 4.0 Research background and limitations
 ++++++++++++++++++++++++++++++++++++++++
 
 .. _MDYLDNote:
 
-4.1 A note on years lived with disability and maternal disorders
+4.1 A note on years lived with disability and maternal  disorders
```
Comment (aflaxman) [inline docs/source/models/concept_models/vivarium_nutrition_optimization/pregnancies/concept_model.rst:677]:
```suggestion
4.1 A note on years lived with disability and maternal disorders
```

**Review by aflaxman — APPROVED** [review]

**Pull request merged**